recipes/ndindex/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ndindex" %}
+{% set version = "1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c309efa54b1a21f18bfd1c9e28aa81456b2c2d6cdf4d7496de4f45f46ff2244f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - numpy
+  run:
+    - python >=3.6
+    - numpy
+
+test:
+  imports:
+    - ndindex
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/ndindex/
+  summary: A Python library for manipulating indices of ndarrays.
```

Comment:
Add a comment linking to the following PR as a reminder to potentially use the upstream license file for future releases:

https://github.com/Quansight/ndindex/pull/2